shiny/ui/_page.py

```diff
@@ -314,6 +309,15 @@ def page_fillable(
         lang=lang,
     )
 
+    # page returns a <html> tag, but we need to make the <body> fillable
+    body = page.children[1]
+    if not isinstance(body, Tag) or body.name != "body":
+        raise ValueError("Expected a <body> tag")
+
+    as_fillable_container(body)
```

Comment:
This function is being called for its side effect, modifying the object in place. I think in all other instances in the code base, we call it for its return value.

The function definition has a question of how it should work:
https://github.com/posit-dev/py-shiny/blob/ddfd7628608ee1bdc834110cd35bae5efb78c3f0/shiny/ui/fill/_fill.py#L33-L40

I think that for clarity and consistency, that function should only be used for its return value, and shouldn't alter the object in place. I'll make a follow-up PR that changes that function, and also alters the usage here to match.